scripts/TileAlgebra.js

```diff
@@ -125,49 +135,62 @@ var tileAlgebra = (function () {
         return success;
     };
 
+    val_tile = function(x,y,z,){}
+
+    let resultset = new Set();
+
     // given the polygon geoJSON, figure out which tiles are within the polygon
     // this may benefit from rewriting - we start with coords, convert to tile indices,
     // then convert each index tuple BACK to coords
     // should this be parallelized or given to workers? If it is very fast compared to sending all the HTTP requests
     // to AWS, then perhaps we just do this in the main thread and just offload the validation
     let filter_tiles = function(NE,SW,z,polygon_gj) {
         const polygon_tf = turf.polygon(polygon_gj.geometry.coordinates);
+        let start_x = _long2tile(SW.lng, z);
         let stop_x  = _long2tile(NE.lng, z);
         let start_y = _lat2tile(NE.lat, z);
-        let start_x = _long2tile(SW.lng, z);
         let stop_y  = _lat2tile(SW.lat, z);
+        console.log(`Running from x: ${start_x} - ${stop_x} and y: ${start_y} - ${stop_y}`);
+        console.log(`That's a total of ${Math.abs((start_x-stop_x+1)*(start_y - stop_y + 1))} tiles to check!`);
```

Comment:
Scott,

Do we want to use console.debug() hear?
Otherwise we will pollute logs ...

Reply:
Sure - I didn't have any coherent assignment of logging levels. Those are better off as debug.